app/controllers/v5/stories_controller.rb

```diff
@@ -0,0 +1,93 @@
+module V5
+  class StoriesController < V5::BaseUserController
+    power :stories, map: {
+      [:show] => :show_stories,
+      [:update] => :update_stories,
+      [:create] => :create_story
+    }, as: :story_scope
+
+    def index
+      load_stories
+      render_stories
+    end
+
+    def show
+      load_story or render_not_found
+      render_story
+    end
+
+    def create
+      if build_story
+        render_story :created
+      else
+        render_errors
+      end
+    end
+
+    def update
+      load_story or render_error('Invalid story id') && return
+      if build_story
+        render_story
+      else
+        render_errors
+      end
+    end
+
+    private
+
+    def load_story
+      @story ||= story_scope.find_by(id: params[:id])
+    end
+
+    def load_stories
+      @story_filter ||= StoryFilter.new(get_params, story_scope)
+    end
+
+    def build_story
+      @story ||= story_scope.new
+      @story.attributes = story_params
+      @story.save
+    end
+
+    def render_story(status = nil)
+      status ||= :ok
+      render json: @story, status: status, serializer: StorySerializer if @story
+    end
+
+    def render_stories
+      render json: @story_filter.filtered,
+             serializer: StoriesSerializer,
+             page: @story_filter.page,
```

Comment:
Yes, any property passed to render that isn't an active model serializer specific option will be added to the instance_options Hash. The StoryFilter deals with page and per_page defaults, so I pass in what it's determined them to be. We could also determine these in the serializer based off the ActiveRecordRelation.limit and .offset, but this seemed to be less confusing.